code/__HELPERS/_macros.dm

```diff
@@ -39,7 +39,7 @@
 
 #define iscluwne(A) istype(A, /mob/living/simple_animal/hostile/retaliate/cluwne)
 
-#define isAI(A) istype(A, /mob/living/silicon/ai)
+#define isAI(A) (istype(A, /mob/living/silicon/ai) || (isobserver(A) && ticker.current_state == GAME_STATE_FINISHED)) 
```

Comment:
Either way the behavior you are trying to obtain is handled by polymorphism in click code. This isn't even likely to help you achieve that.